openshift-client/src/main/java/io/fabric8/openshift/client/dsl/internal/TemplateOperationsImpl.java

```diff
@@ -214,4 +220,22 @@ private URL getProcessUrl() throws MalformedURLException {
     return new TemplateOperationsImpl(client, OpenShiftConfig.wrap(config), null, namespace, null, false, template, null, false, 0L,
       null, null, null, null, null);
   }
+
+  @Override
+  protected <T> T unmarshal(Class<T> type, InputStream is) throws IOException {
+    if (replaceValueStream != null) {
```

Comment:
Apparently, I missing something here. `load()` is a method which I expect it to be called before `processLocally()` (which is the method that sets value to `replaceValueStream`) . So I don't see how its possible this not to be null.